app/assets/stylesheets/coordinator.css.scss

```diff
@@ -160,15 +160,22 @@
                 border-bottom: 1px solid $dark-grey;
                 color: $charcoal;
                 .donation-info {
+                    @extend text-center;
+
                     h1.donation-info-title {
-                        @extend .text-center;
                         color: $charcoal;
                         padding-top: 20px;
                         padding-bottom: 20px;
                         font-size: rem-calc(24);
                     }
-                    h2.donation-info-datetime {
-                        font-size: rem-calc(18);
+                    img.coordinator-donation-picture {
```

Comment:
Avoid qualifying class selectors with an element.<br>Selector should have depth of applicability no greater than 2, but was 6